src/SearchResult.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const SearchResult = props => {
+  const [isSelected, setSelected] = useState(false);
+
+  const handleClick = () => {
+    setSelected(selected => !selected);
+  };
```

Comment:
This is excellent work! A lot of people struggle with this exercise because they don't think about extracting `SearchResult` as a separate component. But it's pretty straightforward to solve if you do 🎆